src/domain/Lotto.js

```diff
@@ -0,0 +1,73 @@
+import utils from '../utils/utils.js';
+import VALUES from '../constants/values.js';
+import MESSAGES from '../constants/messages.js';
+
+class Lotto {
+  static #FORMAT = /^\d+(,\d+)*$/;
+
+  static #MESSAGE = {
+    format: '[ERROR] 쉼표와 숫자를 제외한 다른 문자를 입력했습니다.',
+    size: '[ERROR] 숫자를 6개 입력하지 않았습니다.',
+  };
+
+  #numbers;
+
+  constructor(numbersInput) {
+    this.#updateNumbers(numbersInput);
+    this.#validate(numbersInput);
+  }
+
+  #updateNumbers(numbersInput) {
+    const numbers = Lotto.#convertStringToNumericArray(numbersInput);
+    this.#numbers = utils.sortRandoms(numbers);
+  }
+
+  static #convertStringToNumericArray(numbersInput) {
+    const numbers = [];
+
+    numbersInput.split(VALUES.separator).forEach((string) => {
+      numbers.push(Number(string));
+    });
+
+    return numbers;
```

Comment:
forEach를 사용하지 않고 map을 사용하면 코드가 더 간결해질 것 같아요.
```suggestion
    return numberInput.split(VALUES.separator).map(Number);
```

Reply:
좋은 지적이십니다! 지금 보니 오? 왜 이렇게 했지?라는 생각이 드네요 😅